build/tasks/docs.js

```diff
@@ -6,45 +6,57 @@ const glob = require('glob');
 const del = require('del');
 const vinylPaths = require('vinyl-paths');
 const docjs = require('documentation');
+const REGEX_FILE_NAME = /((\w)+(-)*(\w))+(?=.js)/g;
```

Comment:
I just moved them up to a global constant. This regex seem to match a file name pattern like this ws-something-somehting.js
Maybe it could be done simpler but there was no need to simplify it right now in my eyes^^